Common/UnitDefinitions/CoefficientOfThermalExpansion.json

```diff
@@ -0,0 +1,46 @@
+﻿{
+  "Name": "CoefficientOfThermalExpansion",
+  "BaseUnit": "InverseKelvin",
+  "XmlDoc": "A unit that represents a fractional change in size in response to a change in temperature.",
+  "BaseDimensions": {
+    "Θ": -1
+  },
+  "Units": [
+    {
+      "SingularName": "InverseKelvin",
+      "PluralName": "InverseKelvin",
```

Comment:
InverseKelvins?

Reply:
I think we should leave them as singular. Coefficients of thermal expansion can be though of as a fractional change "per Kelvin" (K⁻¹ or 1/K).

The unit is always in relation to a change of 1 K, °C, or °F.